src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedInstanceMethodSymbol.cs

```diff
@@ -76,5 +76,7 @@ internal override int CalculateLocalSyntaxOffset(int localPosition, SyntaxTree l
         internal override bool IsInitOnly => false;
 
         public sealed override FlowAnalysisAnnotations FlowAnalysisAnnotations => FlowAnalysisAnnotations.None;
+
+        internal override bool IsNullableAnalysisEnabled() => false;
```

Comment:
Feels like this should remain abstract.

Reply:
There are many derived types, and the interesting implicit constructors do not derive from this type, so I think I'd rather keep this as is.

---
In reply to: [551489410](https://github.com/dotnet/roslyn/pull/49953#discussion_r551489410) [](ancestors = 551489410)